opendebates/moderator_views.py

```diff
@@ -60,6 +60,13 @@ def merge(request):
         votes_to_merge = Vote.objects.filter(submission=to_remove).exclude(
             voter__in=votes_already_cast)
 
+        previous_debate_time = get_previous_debate_time()
+        if previous_debate_time:
+            current_votes_to_merge = votes_to_merge.filter(
+                created_at__gt=previous_debate_time).count()
+        else:
+            current_votes_to_merge = 0
```

Comment:
This seems contrary to the behavior of `current_votes`, which is incremented even if `previous_debate_time is None` (which as noted elsewhere I'm not sure is needed).